apps/remix-ide-e2e/src/tests/debugger.test.ts

```diff
@@ -183,6 +183,7 @@ module.exports = {
     .getEditorValue((content) => {
       browser.assert.ok(content.indexOf('if slt(sub(dataEnd, headStart), 32) { revert(0, 0) }') != -1, 'current displayed content is not a generated source')
     })
+    .modalFooterOKClick()
```

Comment:
why do you need to do that?

Reply:
I had to because of error of fetching. now I'll remove it